pkgs/development/ocaml-modules/bls12-381-signature/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  fetchzip,
+  buildDunePackage,
+  bls12-381,
+  alcotest,
+  bisect_ppx,
+  integers_stubs_js,
+}:
+
+buildDunePackage rec {
+  pname = "bls12-381-signature";
+  version = "1.0.0";
+  src = fetchzip {
+    url = "https://gitlab.com/nomadic-labs/cryptography/ocaml-${pname}/-/archive/${version}/ocaml-bls12-381-signature-$¥{version}.tar.bz2";
```

Comment:
This is unusual: `$¥{`. What is it?

Reply:
That's a typo, I guess it only works because I had the cache before I added the variables

Reply:
Fixed